hepcrawl/extractors/iop_parser.py

```diff
@@ -95,22 +101,25 @@ def parse_node(self, response, node):
         # local file paths
         local_files = []
         if 'xml_url' in response.meta:
-            local_files.append({'filetype': 'xml', 'path': response.meta['xml_url']})
+            local_files.append(
+                {'filetype': 'xml', 'path': response.meta['xml_url']})
         if 'pdf_url' in response.meta:
-            local_files.append({'filetype': 'pdf', 'path': response.meta['pdf_url']})
+            local_files.append(
+                {'filetype': 'pdf', 'path': response.meta['pdf_url']})
         if 'pdfa_url' in response.meta:
-            local_files.append({'filetype': 'pdf/a', 'path': response.meta['pdfa_url']})
+            local_files.append(
+                {'filetype': 'pdf/a', 'path': response.meta['pdfa_url']})
         record.add_value('local_files', local_files)
 
         return dict(record.load_item())
 
     def get_arxiv_eprints(self, node):
         arxiv_eprints = []
-
-        arxivs_raw = node.xpath("//custom-meta-group/custom-meta/meta-value/text()")
+        arxivs_raw = node.xpath(
+            "//custom-meta-group/custom-meta/meta-value/text()")
         for arxiv in arxivs_raw:
-            ar = arxiv.extract().replace('arXiv:', '')
-            if ar:
-                arxiv_eprints.append({'value': ar})
+            arxiv_value = arxiv.extract()
+            pattern = re.compile(r'(ar[X-x]iv:|v[0-9]$)')
```

Comment:
no need to do this. In regex there is an option for that https://docs.python.org/3/library/re.html#re.IGNORECASE

Reply:
The compile should not be inside the loop, since if we add it there it will compile all the time